src/N3DataFactory.js

```diff
@@ -63,6 +64,12 @@ export class Term {
 
 // ## NamedNode constructor
 export class NamedNode extends Term {
+  constructor(id, token) {
+    super(id);
+
+    this.token = token;
+  }
+
```

Comment:
That's OK. So will the default be context = { token = null } or context = null? 

I am personally hesitant to initialize `token` with an empty string as this is a different datataype as the `Token` object and it might indicate that one should expect a non-empty string when it is initalized which is misleading. 

But I'll do as you wish. Just let me know.

Reply:
OK. Saw the answer in the comment below.